app/src/main/java/com/github/se/cyrcle/model/user/User.kt

```diff
@@ -1,5 +1,7 @@
 package com.github.se.cyrcle.model.user
 
+const val MAX_NOTE_LENGTH = 48
```

Comment:
Seems a little short to me. Non-blocking of course

Reply:
Mhhh, I conceived note to be an "Alternative title", one that is more familiar to the user. This is way I chose to make it as long as the title